### PR TITLE
Adds a projects section. Updates hero image. Adds icons & links

### PR DESCRIPTION
**Pull request by hailed748**
Adds a project section to the web page which features a slideshow of images. Also updates the hero image to take up the whole page. Lastly, adds GitHub and LinkedIn links in the top right corner.

### CODE REVIEW

**Comment by LukeGMTravis** [comment]
@Tetheta Starting review

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -9,16 +9,20 @@
   <body>
     <div class="jumbotron">
       <h1 class="headline-text">Dagmawi Haile</h1>
+      <div class = "social">
```
Comment (LukeGMTravis) [inline portfolio/src/main/webapp/index.html:12]: Remove extra whitespace

File: portfolio/src/main/webapp/script.js [hunk 2]
```diff
@@ -12,17 +12,26 @@
...
-/**
- * Adds a random greeting to the page.
- */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
 
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
+const pictures = ["images/proj1.jpg","images/proj2.jpg","images/proj3.jpg"]
```
Comment (LukeGMTravis) [inline portfolio/src/main/webapp/script.js:16]: Add spaces after commas

File: portfolio/src/main/webapp/script.js [hunk 3]
```diff
@@ -12,17 +12,26 @@
...
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
+function nextImage(){
+    if(currentIndex < pictures.length - 1){
+        currentIndex ++;
+    }
+    else{
+        currentIndex = 0;
+    }
+    document.slide.src = pictures[currentIndex]
```
Comment (LukeGMTravis) [inline portfolio/src/main/webapp/script.js:26]: I actually didn't realize you could do this. However, conventionally html elements are accessed by id or class only so it's probably better to switch to getting the element that way.

File: portfolio/src/main/webapp/script.js [hunk 4]
```diff
@@ -12,17 +12,26 @@
...
-  // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
+function nextImage(){
+    if(currentIndex < pictures.length - 1){
+        currentIndex ++;
+    }
+    else{
+        currentIndex = 0;
+    }
```
Comment (LukeGMTravis) [inline portfolio/src/main/webapp/script.js:25]: While this works, a (potentially cleaner) alternative is to increment and then use mod division to get the new index. Up to you which way you like better.

File: portfolio/src/main/webapp/style.css [hunk 5]
```diff
@@ -1,55 +1,71 @@
...
-
 body,
 html {
     margin: 0;
     padding: 0;
 }
 
+.jumbotron {
+    align-items: center;
+    background-image: linear-gradient(rgba(0,0,0,0.7),rgba(0,0,0,0.7)),url("/images/jumbo4.jpg");
```
Comment (LukeGMTravis) [inline portfolio/src/main/webapp/style.css:9]: Linting: Spaces after commas

File: portfolio/src/main/webapp/style.css [hunk 6]
```diff
@@ -64,14 +80,40 @@ html {
...
-.jumbotron {
+#about-title {
+    font: 32px Arial, Helvetica, sans-serif;
+    font-weight: bold;
+    text-align: center;
+    text-decoration: underline;
+}
+
+.about-body{
+    font: 16px/2 Arial, Helvetica, sans-serif;
```
Comment (LukeGMTravis) [inline portfolio/src/main/webapp/style.css:91]: This is neat with the font shorthand, I haven't seen this before

Comment (hailed748) [reply portfolio/src/main/webapp/style.css:91]: Yah, I actually learned it from the style guide.

File: portfolio/src/main/webapp/style.css (hunk 5 above)
Comment (hailed748) [reply portfolio/src/main/webapp/style.css:9]: Got it. Spacing is probably the biggest adjustment for me in regards to the style guide, it's easy to forget until its a habit.

File: portfolio/src/main/webapp/script.js (hunk 3 above)
Comment (hailed748) [reply portfolio/src/main/webapp/script.js]: Will do.

File: portfolio/src/main/webapp/script.js (hunk 4 above)
Comment (hailed748) [reply portfolio/src/main/webapp/script.js]: I realized that this code could actually be done better. I fixed this in my update.

File: portfolio/src/main/webapp/index.html [hunk 7]
```diff
@@ -9,7 +9,7 @@
   <body>
     <div class="jumbotron">
       <h1 class="headline-text">Dagmawi Haile</h1>
-      <div class = "social">
+      <div class= "social">
```
Comment (LukeGMTravis) [inline portfolio/src/main/webapp/index.html:12]: Still an extra space after the equals sign

File: portfolio/src/main/webapp/script.js [hunk 8]
```diff
@@ -13,25 +13,15 @@
...
-    if(currentIndex > 0){
-        currentIndex --;
-    }
-    else{
-        currentIndex = pictures.length - 1;
-    }
-    document.slide.src = pictures[currentIndex]
-}
\ No newline at end of file
+    currentIndex = ((currentIndex - 1  % pictures.length) + pictures.length) % pictures.length;
```
Comment (LukeGMTravis) [inline portfolio/src/main/webapp/script.js:25]: You could simplify this even further to `(currentIndex + pictures.length - 1) % pictures.length`

File: portfolio/src/main/webapp/style.css [hunk 9]
```diff
@@ -6,7 +6,7 @@ html {
 
 .jumbotron {
     align-items: center;
-    background-image: linear-gradient(rgba(0,0,0,0.7),rgba(0,0,0,0.7)),url("/images/jumbo4.jpg");
+    background-image: linear-gradient(rgba(0, 0, 0, 0.7),rgba(0, 0, 0, 0.7)),url("/images/jumbo4.jpg");
```
Comment (LukeGMTravis) [inline portfolio/src/main/webapp/style.css:9]: Sorry to be nit-picky but don't forget to put spaces after the other commas as well

Comment (hailed748) [reply portfolio/src/main/webapp/style.css]: No problem, hopefully I got all of the style changes this time.

**Pull request merged**